service/lib/dinstaller/dbus/users_service.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "dbus"
+require "dinstaller/dbus/users"
+
+module DInstaller
+  module DBus
+    # D-Bus service (org.opensuse.DInstaller)
+    #
+    # It connects to the system D-Bus and answers requests on objects below
+    # `/org/opensuse/DInstaller/Users`.
+    class UsersService
```

Comment:
```
 service/lib/dinstaller/dbus/clients/users.rb
 service/lib/dinstaller/dbus/users.rb
 service/lib/dinstaller/dbus/users_service.rb
```

This is getting complicated, I'll think about naming, namespacing and commenting the architecture so that it makes sense for newcomers